android/src/com/frostwire/android/gui/fragments/BrowsePeerFragment.java

```diff
@@ -528,126 +533,92 @@ public Object loadInBackground() {
         loader.forceLoad();
         return loader;
     }
-
+    
     private void updateHeader() {
+        updateHeader(true);
```

Comment:
wonder if here you should not do the `updateFinger=true` if you're not in normal state.